LICENSE

```diff
@@ -1,4 +1,6 @@
-Copyright (c) 2014-2015 Vokal 
+https://tldrlegal.com/license/mit-license
```

Comment:
If you look at the full text there, they have "The MIT License (MIT)" on the first line...